content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -0,0 +1,111 @@
+
+When deploying a VM using [`istioctl workload entry configure`](/docs/setup/install/virtual-machine/), basic DNS proxying will be enabled by default.
+
+## DNS Capture In Action
+
+To try out the DNS capture, first setup a `ServiceEntry` for some external service.
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1beta1
+kind: ServiceEntry
+metadata:
+  name: external-address
+spec:
+  addresses:
+  - 198.51.100.0
+  hosts:
+  - address.internal
+  ports:
+  - name: http
+    number: 80
+    protocol: HTTP
+{{< /text >}}
+
+Without the DNS capture, a request to `address.internal` would likely fail to resolve. Once this is enabled, we should instead get a response back based on the configured `address`:
+
+{{< text bash >}}
+$ curl -v address.internal
+*   Trying 198.51.100.1:80...
+{{< /text >}}
+
+## Address Auto Allocation
+
+In the example above, we had a predefined IP address for the service that we wanted to send. However, its common to access external services which do not have stable addresses, and instead rely on DNS. In this case, the DNS proxy will not have enough information to return a response, and will need to forward DNS requests upstream.
+
+This is especially problematic with TCP traffic. Unlike HTTP requests, which are routed based on `Host` headers, TCP carries much less information; we can only route on the destination IP and port number. Because we do not have a stable IP for the backend, we cannot route based on that either, leaving only port number, which leads to conflicts when multiple `ServiceEntry`s for TCP services share the same port.
+
+To work around these issues, the DNS proxy additionally supports automatically allocating addresses for `ServiceEntry`s that do not explicitly define one. This is configured by the `ISTIO_META_DNS_AUTO_ALLOCATE` option mentioned above.
+
+When this feature is enabled, the DNS response will include a distinct, automatically assigned address for each `ServiceEntry`. The proxy is then configured to match requests to this IP address, and forward the request to the corresponding `ServiceEntry`.
+
+{{< warning >}}
+Because this feature modifies DNS responses, it may not be compatible with all applications.
+{{< /warning >}}
+
+To try this out, configure another `ServiceEntry`:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1beta1
+kind: ServiceEntry
+metadata:
+  name: external-auto
+spec:
+  hosts:
+  - auto.internal
+  ports:
+  - name: http
+    number: 80
+    protocol: HTTP
+  resolution: STATIC
+  endpoints:
+  - address: 198.51.100.2
+{{< /text >}}
+
+Now, send a request:
+
+{{< text bash >}}
+$ curl -v auto.internal
+*   Trying 240.240.0.1:80...
+{{< /text >}}
+
+We can see the request is sent to an automatically allocated address, `240.240.0.1`. These address will be picked from the `240.240.0.0/16` reserved IP address range to avoid conflicting with real services.
```

Comment:
```suggestion
As you can see, the request is sent to an automatically allocated address, `240.240.0.1`. These addresses will be picked from the `240.240.0.0/16` reserved IP address range to avoid conflicting with real services.
```